pkg/domain/infra/abi/play.go

```diff
@@ -453,12 +453,17 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 
 	containerConfig.Command = []string{}
 	if imageData != nil && imageData.Config != nil {
-		containerConfig.Command = append(containerConfig.Command, imageData.Config.Entrypoint...)
+		containerConfig.Command = imageData.Config.Entrypoint
 	}
 	if len(containerYAML.Command) != 0 {
-		containerConfig.Command = append(containerConfig.Command, containerYAML.Command...)
-	} else if imageData != nil && imageData.Config != nil {
-		containerConfig.Command = append(containerConfig.Command, imageData.Config.Cmd...)
+		containerConfig.Command = containerYAML.Command
+	}
+	if imageData != nil && imageData.Config != nil {
+		if len(containerYAML.Args) != 0 {
```

Comment:
we shouldn't have this block here. it should be outside of the imageData block. We should set it to containerYAML.Args regardless of whether imageData is defined.

Reply:
ok,I will change it